userdetails-gorm/src/main/groovy/au/org/ala/userdetails/gorm/GormApplicationService.groovy

```diff
@@ -133,6 +134,7 @@ class GormApplicationService implements IApplicationService {
         if (applicationRecord.type == ApplicationType.GALAH) {
             callbacks.addAll(galahCallbackURLs)
         }
+        callbacks.addAll(tokensCallbackURLs)
```

Comment:
I'm not sure we should be adding tokens callback URLs to all apps? Maybe add a checkbox to allow the user to opt-in?

Reply:
Fixed